pytest_reana/fixtures.py

```diff
@@ -154,6 +154,41 @@ def test_default_user_exists(default)
     return user
 
 
+@pytest.fixture()
+def second_user(app, session):
+    """Create users.
+
+    Scope: function
+
+    This fixture creates a user with UUID
+    ``11111111-1111-1111-1111-111111111111``, ``email`` `info2@reana.io`
```

Comment:
BTW please rebase against latest `master`, and you could also prepare a release commit, because we would need to release a new version and in depending PRs we would need to ask for higher version of `pytest-package`, otherwise the PRs would not pass (for your workflow sharing features), but also other components would start to fail (because there would be no `default_user` anymore).

Reply:
Did the rebase. I am not sure how to do a release commit though